folly/Range.h

```diff
@@ -247,10 +245,11 @@ class Range {
   using size_type = std::size_t;
   using iterator = Iter;
   using const_iterator = Iter;
-  using value_type = typename std::remove_reference<
-      typename std::iterator_traits<Iter>::reference>::type;
-  using difference_type = typename std::iterator_traits<Iter>::difference_type;
   using reference = typename std::iterator_traits<Iter>::reference;
+  using const_reference = typename std::add_const<reference>::type;
```

Comment:
For new changes let’s use the C++17 style traits.

Reply:
More than happy to change this, I was just sticking to current style of the code.